### ISSUE TITLE
debug mode

### ISSUE THREAD

**Issue by vamzz1** [masumsoft/express-cassandra#26]
Hi there, is there anyway we can run cassandra in debug mode through the orm ??

**Comment by masumsoft** [comment]
Could you please explain what did you mean by debug mode? The cassandra instance runs independently of the orm. This thing should work whether your cassandra instance is running in debug mode or not. The orm communicates with the cassandra cluster to exchange information and data, it does not control the cluster or it's state.

**Comment by vamzz1** [comment]
Debug mode as in the orm will log all the db calls being made.

**Comment by masumsoft** [comment]
Hmm, there is no debug mode as of now, but I agree that would be a nice to have and adding this feature to the orm should be straight forward. Hopefully I'll be working on it this weekend.

**Comment by vamzz1** [comment]
Cool man.Thanks

**Comment by masumsoft** [comment]
For now, you may want to use the `return_query` option to get the final built query instead of executing it:

```js
var event = new models.instance.Event({
    id: 3,
    body: 'hello3'
});
var save_query = event.save({return_query: true});
console.log(save_query);

var update_query = models.instance.Event.update(
    {id: 1},
    {body: 'hello1 updated'},
    {return_query: true}
);
console.log(update_query);

var delete_query = models.instance.Event.delete(
    {id: 2},
    {return_query: true}
);
console.log(delete_query);
```

**Comment by masumsoft** [comment]
debug messages are now available in the latest version (0.7.0), please refer to the doc: https://github.com/masumsoft/express-cassandra#debug-logging-queries
